pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry-core>=1.9.0"]
+requires = ["poetry-core>=1.0.0"]
```

Comment:
Potential fix for `Unable to find installation candidates for rpds-py (0.18.1)` - recurring issue in dev docs CI